sklearn/gaussian_process/gaussian_process.py

```diff
@@ -566,20 +565,14 @@ def reduced_likelihood_function(self, theta=None):
             A dictionary containing the requested Gaussian Process model
             parameters:
 
-                sigma2
```

Comment:
Apart from being over indented what's the problem here? Isn't this otherwise a valid rst definition list?